testbed/addons/karpenter/construct.ts

```diff
@@ -4,24 +4,25 @@ import iam = require('@aws-cdk/aws-iam');
 
 export interface KarpenterProps {
     cluster: eks.Cluster;
-    namespace?: string;
```

Comment:
I actually thought it was nice that we could configure this.

Reply:
Spoke offline, leaning to convention here than configurability. Same for following other comments on this PR where applicable.